frameworks/Scala/zio-http/src/main/scala/Main.scala

```diff
@@ -0,0 +1,41 @@
+import zhttp.http._
+import zhttp.service.Server
+import zio.{App, ExitCode, URIO}
+import io.netty.handler.codec.http.{HttpHeaderNames => JHttpHeaderNames, HttpHeaderValues => JHttpHeaderValues}
+
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+
+object WebApp extends App {
+
+  val message: String = "Hello, World!"
+def createDate: String = DateTimeFormatter.RFC_1123_DATE_TIME.format(ZonedDateTime.now)
+  import com.github.plokhotnyuk.jsoniter_scala.macros._
+  import com.github.plokhotnyuk.jsoniter_scala.core._
+
+  case class Message(message: String)
+  implicit val codec: JsonValueCodec[Message] = JsonCodecMaker.make
+
+  val jsonstr = new String(writeToArray(Message(message)))
+
+  val app = Http.collect[Request] {
+    case Method.GET -> Root / "plaintext" =>  Response.http(
+      headers = List(
+        Header(JHttpHeaderNames.SERVER, "zio-http"),
+        Header(JHttpHeaderNames.DATE, s"$createDate"),
+        Header(JHttpHeaderNames.CONTENT_TYPE, JHttpHeaderValues.TEXT_PLAIN),
+      ),
+      content = HttpContent.Complete("Hello, World!"),
+    )
+    case Method.GET -> Root / "json" => Response.http(
+      headers = List(
+        Header(JHttpHeaderNames.SERVER, "zio-http"),
+        Header(JHttpHeaderNames.DATE, s"$createDate"),
+        Header(JHttpHeaderNames.CONTENT_TYPE, JHttpHeaderValues.APPLICATION_JSON),
+      ),
+      content = HttpContent.Complete(jsonstr))
+  }
+
+  override def run(args: List[String]): URIO[zio.ZEnv, ExitCode] = Server.start(8080, app).exitCode
```

Comment:
Suggestion: disable zio tracing :)

Also, thread count should probably match the one in other frameworks (https supposedly has 32: https://github.com/TechEmpower/FrameworkBenchmarks/blob/a3771b148f434b007fe6d63f97007cdbc2c145ff/frameworks/Scala/http4s/src/main/scala/http4s/techempower/benchmark/WebServer.scala#L98)

Reply:
By default AFAIK it's going to be 2x `availableCPU()`. Need to check if the test machine already has virtualization because in that case it should be equal to `availableCPU()`